app/controllers/TestTaskController.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+class TasksController extends \BaseController {
+
+	/**
+	 * Display a listing of the resource.
+	 * GET /tasks
+	 *
+	 * @return Response
+	 */
+	public function index()
+	{
+		$pTitle		=	"Tasks";
+
+		$user 		= 	User::find(Auth::id());		
+		$tasks 		= 	$user->tasks()->whereState("incomplete")->get();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "incomplete" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://172.27.59.109:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused "$tasks" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://172.27.59.109:9000/coding_rules#rule_key=php%3AS1481)